tapestry.js

```diff
@@ -41,47 +30,46 @@ function tapestryTool(config){
         TIME_BETWEEN_SAVE_PROGRESS = 5, // Means the number of seconds between each save progress call
         NODE_UNLOCK_TIMEFRAME = 2, // Time in seconds. User should be within 2 seconds of appearsAt time for unlocked nodes
         API_PUT_METHOD = 'PUT',
+        API_DELETE_METHOD = 'DELETE',
         USER_NODE_PROGRESS_URL = config.apiUrl + "/users/progress",
         USER_NODE_UNLOCKED_URL = config.apiUrl + "/users/unlocked",
         TAPESTRY_H5P_SETTINGS_URL = config.apiUrl + "/users/h5psettings";
 
+
     var // declared variables
         root, svg, links, nodes,                        // Basics
         path, pieGenerator, arcGenerator,               // Donut
-        simulation,                                     // Force
+        linkForce, collideForce, force,                 // Force
```

Comment:
Why are you changing these things? This was changed recently and merged into master and also into the use-vue-js branch. Please be careful not to change things unless they are related to this issue (i.e. deleting nodes and links). Please go through your code and revert any changes that you had made that would undo changes that were merged in already not related to your issue.